instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -80,6 +80,7 @@ main()
     mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/lib/ext/twofactorauth
     mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/lib/ext/nginx-lookup
     mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/lib/ext/openidconsumer
+    mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/lib/ext/network
     mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/lib/ext/zimbra-license
```

Comment:
Are we missing copying of the jar from the build dir.

Reply:
ok, fixed now. Thanks